### ISSUE TITLE
Build failed due to a user error: Build script returned non-zero exit code: 2

### ISSUE THREAD

**Issue by ax-6** [aq-org/Docs#13]
```
3:36:38 PM: Netlify Build                                                 
3:36:38 PM: ────────────────────────────────────────────────────────────────
3:36:38 PM: ​
3:36:38 PM: ❯ Version
3:36:38 PM:   @netlify/build 29.53.0
3:36:38 PM: ​
3:36:38 PM: ❯ Flags
3:36:38 PM:   accountId: 644b9fef3eaa21009214dded
3:36:38 PM:   baseRelDir: true
3:36:38 PM:   buildId: 66c598d65298a1000827c848
3:36:38 PM:   deployId: 66c598d65298a1000827c84a
3:36:38 PM: ​
3:36:38 PM: ❯ Current directory
3:36:38 PM:   /opt/build/repo
3:36:38 PM: ​
3:36:38 PM: ❯ Config file
3:36:38 PM:   No config file was defined: using default values.
3:36:38 PM: ​
3:36:38 PM: ❯ Context
3:36:38 PM:   production
3:36:38 PM: ​
3:36:38 PM: Build command from Netlify app                                
3:36:38 PM: ────────────────────────────────────────────────────────────────
3:36:38 PM: ​
3:36:38 PM: $ npm run build
3:36:38 PM: > build
3:36:38 PM: > astro check && astro build
3:36:40 PM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
3:36:40 PM: ​
3:36:40 PM: "build.command" failed                                        
3:36:40 PM: ────────────────────────────────────────────────────────────────
3:36:40 PM: ​
3:36:40 PM:   Error message
3:36:40 PM:   Command failed with exit code 1: npm run build (https://ntl.fyi/exit-code-1)
3:36:40 PM: ​
3:36:40 PM:   Error location
3:36:40 PM:   In Build command from Netlify app:
3:36:40 PM:   npm run build
3:36:40 PM: ​
3:36:40 PM:   Resolved config
3:36:40 PM:   build:
3:36:40 PM:     command: npm run build
3:36:40 PM:     commandOrigin: ui
3:36:40 PM:     publish: /opt/build/repo/dist
3:36:40 PM:     publishOrigin: ui
3:36:40 PM: Build failed due to a user error: Build script returned non-zero exit code: 2
3:36:40 PM: Failing build: Failed to build site
3:36:40 PM: Finished processing build request in 18.289s
```

**Comment by ax-6** [comment]
#12 ERROR

**Comment by ax-6** [comment]
The problem is in astro.config.mjs. To be more precise, it is in the new additions under defineConfig, integrations, starlight, social.

**Comment by ax-6** [comment]
> The problem is in astro.config.mjs. To be more precise, it is in the new additions under defineConfig, integrations, starlight, social.

This was discovered by deploying in a local environment and then deploying it after commenting out the additions. Looking for the code that has the problem.

**Comment by ax-6** [comment]
Here is the code (the commented out area is what causes the error):
```
import { defineConfig } from 'astro/config';
import starlight from '@astrojs/starlight';

// https://astro.build/config
export default defineConfig({
	integrations: [
		starlight({
			title: 'AQ Docs',
			social: {
				github: 'https://github.com/aq-org/AQ',
				gitlab: 'https://gitlab.com/aqorg/AQ',
				'x.com': 'https://x.com/aq_organization',
				twitter: 'https://twitter.com/aq_organization',
				youtube: 'https://www.youtube.com/@aq-org',
				facebook: 'https://www.facebook.com/aq.organization',
				email: 'mailto:admin@axa6.com',
				discord: 'https://discord.gg/JCKzxavG',
				twitch: 'https://www.twitch.tv/axa6t',
				/*bitbucket: 'https://bitbucket.org/ax-6/aq',
				bluesky: 'https://bsky.app/profile/aq-org.bsky.social',
				codeberg: 'https://codeberg.org/aq-org/AQ',
				openCollective: 'https://opencollective.com/aq-org',
				mastodon: 'https://mastodon.social/@aqorg',
				patreon: 'https://www.patreon.com/AQORG',
				reddit: 'https://www.reddit.com/r/aq_organization',
				slack: 'https://aq-organization.slack.com',
				stackOverflow: 'https://stackoverflow.com/users/25469681/aq-org',*/
			},
		}),
	],
});

```

**Comment by ax-6** [comment]
By comparing with [the official Starlight documents](https://starlight.astro.build/zh-cn/reference/configuration/#social), we found that the cause of the problem is `bluesky: 'https://bsky.app/profile/aq-org.bsky.social'`, the `bluesky` should be `blueSky`. We are trying to fix the error.

**Comment by ax-6** [comment]
This issue has been fixed: #14
